### PR TITLE
serpy post

### PR DESCRIPTION
**Pull request by clarkduvall**

### CODE REVIEW

File: _posts/2015-09-04-ditching-drf-serializers-for-serpy.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - A lot of obvious work being done in Python? __NOPE__
+
+A typical view I was seeing this pattern for looked something like this:
+
+```py
+class GoalsView(BaseView):
+
+    def get(self, user_id, *args, **kwargs):
+      goals = Goal.objects.filter(owner_id=user_id)
+      return Response(GoalSerializer(goals, many=True).data)
```
Comment (diwu1989) [inline _posts/2015-09-04-ditching-drf-serializers-for-serpy.md]: i think lines 32 and 33 need to be indented another 2 spaces
or else people are gonna pep8 us

File: _posts/2015-09-04-ditching-drf-serializers-for-serpy.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+This graph shows a simple benchmark between serpy, DRF serializers, and Marshmallow (another popular
+serialization framework):
+
+![serpy benchmark](/public/img/serpy/benchmark.png)
+
+More benchmarks can be found [here](http://serpy.readthedocs.org/en/latest/performance.html).
+
+We have been using serpy in production for a few months now, and haven't had any more bottlenecks on
+serialization. Django REST Framework is still used for other parts of our app, but serialization
+required a more specialized library.
```
Comment (diwu1989) [inline _posts/2015-09-04-ditching-drf-serializers-for-serpy.md]: instead of `specialized` call `performant`

File: _posts/2015-09-03-sqlalchemy-and-django.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: post
 title: SQLAlchemy and Django
 author: Rebecca
 ---
-At [BetterWorks](http://betterworks.com), we use the [Django ORM](https://docs.djangoproject.com/en/1.8/topics/db) for data persistence and have found it to be very convenient for modeling and basic querying. It's very easy to use and does a lot of the work for you when it comes to dealing with multiple tables in the database. However, it falls short for some of the more complex queries.
+At [BetterWorks](https://betterworks.com), we use the [Django ORM](https://docs.djangoproject.com/en/1.8/topics/db) for data persistence and have found it to be very convenient for modeling and basic querying. It's very easy to use and does a lot of the work for you when it comes to dealing with multiple tables in the database. However, it falls short for some of the more complex queries.
```
Comment (diwu1989) [inline _posts/2015-09-03-sqlalchemy-and-django.md:6]: the http vs https isn't super necessary because we auto-redirect the users to https

**Comment by diwu1989** [comment]
lgtm thanks

File: _posts/2015-09-04-ditching-drf-serializers-for-serpy.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Ditching DRF Serializers for Serpy
```
Comment (diwu1989) [inline _posts/2015-09-04-ditching-drf-serializers-for-serpy.md]: might want to just call this out as `Ditching Django Rest Framework Serializers for Serpy`
its better SEO to spell the project out in full
likewise, change the file name

File: _posts/2015-09-04-ditching-django-rest-framework-serializers-for-serpy.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+too many queries, or the queries are inefficient. These problems are usually pretty easy to solve.
+Other times, our Python code is the problem.
+
+As I was profiling some of our slower APIs, I went through my checklist:
+
+  - Too many SQL queries happening? __NOPE__
+  - Slow SQL queries happening? __NOPE__
+  - A lot of obvious work being done in Python? __NOPE__
+
+A typical view I was seeing this pattern for looked something like this:
```
Comment (Kenadia) [inline _posts/2015-09-04-ditching-django-rest-framework-serializers-for-serpy.md:26]: This phrasing is a little hard to parse, esp. for people who don't immediately know you're talking about Django views
